x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -449,6 +449,8 @@ static RoleDescriptor kibanaSystem(String name) {
                         "read",
                         "index",
                         "delete",
+                        // Require "delete_index" to perform ILM policy actions
+                        TransportDeleteIndexAction.TYPE.name(),
```

Comment:
For my own knowledge, is this different from the `delete_index` named privilege?

It might be my limited understanding of the issue and how we manage these indices, but...
- do we need `delete_index` also on the above indices (`logs-cloud_security_posture.findings-*`)?
- and should we really be applying the same privileges to `vulnerabilities_latest-default*` here? Or is this unnecessary?